frontend/app/components/people-skill-show.js

```diff
@@ -36,27 +42,28 @@ export default Component.extend({
       "Expert"
     ];
     return levelNames[this.get("peopleSkill.level")];
-  }),
+  }
 
-  levelChanged: observer("peopleSkill.level", function() {
+  @observes("peopleSkill.level")
+  levelChanged() {
     this.set("levelValue", this.get("peopleSkill.level"));
-  }),
+  }
 
-  actions: {
-    changePerson(person) {
-      person.then(person => {
-        person.reload().then(person => {
-          this.get("router").transitionTo("person.skills", person.get("id"));
-        });
+  @action
+  changePerson(person) {
+    person.then(person => {
+      person.reload().then(person => {
+        this.get("router").transitionTo("person.skills", person.id);
       });
-    },
+    });
+  }
 
-    hasChanged() {
-      if (!this.get("peopleSkill.level")) {
-        this.sliderHandle = this.$(".slider-handle:first");
-        this.sliderHandle.removeClass("slider-handle");
-        this.$(".in-selection").removeClass("in-selection");
-      }
+  @action
```

Comment:
is this really an action ?

Reply:
It is but the naming is awful. Going to rename that